src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService usr = new UserService();
+        User usrdata = new User(email, password);
```

Comment:
why do we need to create this User?

Reply:
I don't need to create this User in that type.
I update my code in a new commit.